queue_model_enumerables_v1.py

```diff
@@ -26,8 +26,8 @@
 EDNARefImages = namedtuple('EDNARefImages', ['FOUR', 'TWO', 'ONE', 'NONE'])
 EDNA_NUM_REF_IMAGES = EDNARefImages(0, 1, 2, 3)
 
-CentringMethod = namedtuple('CentringMethod', ['MANUAL', 'LOOP', 'FULLY_AUTOMATIC', "NO"])
-CENTRING_METHOD = CentringMethod(0, 1, 2, 3)
+CentringMethod = namedtuple('CentringMethod', ['NO', 'MANUAL', 'LOOP', 'FULLY_AUTOMATIC', 'XRAY'])
+CENTRING_METHOD = CentringMethod(0, 1, 2, 3, 4)
```

Comment:
Hi Ivars,
It seems this changes the numbers in the enum: 0 is now "NO" whereas it was "MANUAL" before ; it can break compatibility (for workflows, if one uses numbers instead of enum constants) so I would be more safe if you could not change the enum... What do you think ?
(I admit it is a little issue, but...)